front/app/utils/dateUtils.ts

```diff
@@ -83,6 +83,27 @@ export function toThreeLetterMonth(date: string, resolution: IResolution) {
 }
 
 export function toFullMonth(date: string, resolution: IResolution) {
```

Comment:
Added this to the tooltip. When grouped by week, the data is stored as the first day of that week. I thought this was a bit misleading so I talked to Yaesul and we decided to show the entire week in the tooltip, e.g. "September 19 - 25 2022" instead of just "September 19 2022"